### ISSUE TITLE
Iris shaders

### ISSUE THREAD

**Issue by SlimF** [AeiouEnigma/sodium-forge#7]
Would it be possible for Iris shaders to be implemented? It would be great to have Sodium levels of performance combined with shaders.

Link to Iris: https://github.com/IrisShaders/Iris

**Comment by AeiouEnigma** [comment]
I haven't looked particularly closely at the Iris project, but its handful of Fabric API dependencies seem simple enough (based on how they're laid out in Iris's build.gradle's depedencies section). If it's simple enough to translate those Fabric functionalities to their Forge equivalents then it should be possible to port Iris to Forge.

A more complicated question though would be whether Forge rendering issues would arise when using the hypothetical Forge port of Iris, and I couldn't begin to guess whether or not such problems would arise.

Basically yeah, I think it's possible, but I don't know how difficult it would end up being. It may very likely be a bit too ambitious for me personally, but I'd at least like to look into it when Iris is in a more finished state.

**Comment by SlimF** [comment]
Iris has gotten its 1.0 release. Don't know how much finished it is compared to one month ago, though.

https://github.com/IrisShaders/Iris/releases/tag/1.0.0

https://www.curseforge.com/minecraft/mc-mods/irisshaders

**Comment by AeiouEnigma** [comment]
Ultimately I don't think I'm the right person to take on the task of porting Iris to Forge, and in light of that determination I am closing this issue. Cheers everyone
